docs/app/services/version/version.service.ts

```diff
@@ -0,0 +1,20 @@
+import { Injectable, Output, EventEmitter } from '@angular/core';
+
+
+@Injectable()
+export class VersionService {
+
+    version: string = 'Angular';
+
+    @Output() versionChange: EventEmitter<string> = new EventEmitter<string>();
```

Comment:
We shouldn't be using `@Output` in a service as this is for components to indicate that they are emitting values. We also shouldn't use and `EventEmitter`, instead you should use a `BehaviorSubject` instead. Currently you have a version variable and an event emitter, a behavior subject is essentially a merge of the two, it stores the value and can be subscribed to so you wont need two separate variables, so you can remove `version`, and rename `versionChange` to `version`.

You should create an enum for the different Angular versions as using a string could cause problems, for example if there is a typo in a string then things may not work as expected.

You should also store the selected version in local storage so if the page is refreshed the version stays the same.